graph/src/util/lfu_cache.rs

```diff
@@ -40,6 +40,12 @@ impl<K, V: Default> CacheEntry<K, V> {
             will_stale: false,
         }
     }
+
+    /// Estimate the size of a `CacheEntry` with the given key and value
+    fn weight(key: &K, value: &V) -> usize {
```

Comment:
Would this be simpler if we just required `K` to impl `CacheWeight` as well?

Reply:
Yeah, I wound up doing that when I fixed the unit test - it is simpler